### PR TITLE
Upgrade to Dropwizard 0.8 final

### PR DESCRIPTION
**Pull request by rtyler**

### CODE REVIEW

File: docs/html5/restapi/report.js [hunk 1]
```diff
@@ -7,11 +7,11 @@ formatter.feature({
   "keyword": "Feature"
 });
 formatter.before({
-  "duration": 4793709591,
```
Comment (GiriDandu) [inline docs/html5/restapi/report.js]: Can you checkout docs directory. It's painful to just go through long diffs

**Comment by GiriDandu** [comment]
:+1:

**Comment by maheshkelkar** [comment]
:+1:

**Comment by rtyler** [comment]
Superseded by #116

**Pull request closed without merging**